### ISSUE TITLE
kubeedge 强制删除image

### ISSUE THREAD

**Issue by sgf0906ggy** [kubeedge/kubeedge#5000]
经过长时间使用测试发现 ，每次edge端得资源 满了以后 所有容器被强制停止 并且容器镜像会被强制删除

**Comment by sgf0906ggy** [comment]
应该和k8s的驱逐机制有关

**Comment by WillardHu** [comment]
> 应该和k8s的驱逐机制有关

Yes, the kubelet will garbage collection the images that are not be reference by the container.
